docs/Contributors_Guide/documentation.rst

```diff
@@ -395,6 +395,20 @@ resulting in the following displayed text:
 
 See `Overleaf's Guide - Matrices <https://www.overleaf.com/learn/latex/Matrices>`_ for more information.
 
+Literal Include
+---------------
+
+This feature is used often in the Release Guide.  But there are other places it can also be used.
```

Comment:
```suggestion
This feature is used often in the Release Guide, but there are other places it could be useful.
```
I am modifying this from the original text we had.  Thank you for putting it in as-is and apologies for the last minute change.